modules/pom.xml

```diff
@@ -22,7 +22,9 @@
         <hibernate.validator.module.xml>${module.xml.basedir}/org/hibernate/validator/module.xml</hibernate.validator.module.xml>
         <hibernate.validator.cdi.module.xml>${module.xml.basedir}/org/hibernate/validator/cdi/module.xml</hibernate.validator.cdi.module.xml>
         <validation.api.module.xml>${module.xml.basedir}/javax/validation/api/module.xml</validation.api.module.xml>
-        <layer.name>hibernate-validator-6.0</layer.name>
+        <layer.name>base</layer.name>
```

Comment:
@gunnarmorling this (very minor) comment still stands. Other than that, it looks good!